data resource nutanix_virtual_machine always results in an error

**Describe the bug**
The data ressource "nutanix_virtual_machine" does not work. the following small example gives an error. 

```
cat main.tf
provider "nutanix" {
  username = "<<redacted>>"
  password = "<<redacted>>"
  endpoint = "<<redacted>>"
  insecure = true
  port     = 9440
}
data "nutanix_virtual_machine" "testvm" {
  vm_id = "1120ba14-de3d-4eef-8fcd-5ed1faea9be4"
}

terraform apply
```


`data.nutanix_virtual_machine.testvm: data.nutanix_virtual_machine.testvm: Invalid address to set: []string{"nic_list", "0", "subnet_reference_name"}`

**Expected behavior**
 "terraform apply" should run without an error

**Logs**
```
data.nutanix_virtual_machine.testvm: Refreshing state...

Error: Error refreshing state: 1 error(s) occurred:

* data.nutanix_virtual_machine.testvm: 1 error(s) occurred:

* data.nutanix_virtual_machine.testvm: data.nutanix_virtual_machine.testvm: Invalid address to set: []string{"nic_list", "0", "subnet_reference_name"}
```

```
2018/10/08 20:08:26 [WARN] Invalid log level: "1". Defaulting to level: TRACE. Valid levels are: [TRACE DEBUG INFO WARN ERROR]
2018/10/08 20:08:26 [INFO] Terraform version: 0.11.8  7a5c1d221ac209bbac66947c369815cd9ca70ed5
2018/10/08 20:08:26 [INFO] Go runtime version: go1.10.1
2018/10/08 20:08:26 [INFO] CLI args: []string{"/export/home/SIMU-LOCAL/simu/bin/terraform", "apply"}
2018/10/08 20:08:26 [DEBUG] Attempting to open CLI config file: /export/home/SIMU-LOCAL/simu/.terraformrc
2018/10/08 20:08:26 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2018/10/08 20:08:26 [INFO] CLI command args: []string{"apply"}
2018/10/08 20:08:26 [INFO] command: empty terraform config, returning nil
2018/10/08 20:08:26 [DEBUG] command: no data state file found for backend config
2018/10/08 20:08:26 [DEBUG] New state was assigned lineage "d1b85439-b453-f8cb-0a53-86f7c9ad9872"
2018/10/08 20:08:26 [INFO] command: backend initialized: <nil>
2018/10/08 20:08:26 [DEBUG] checking for provider in "."
2018/10/08 20:08:26 [WARN] found legacy provider "terraform-provider-nutanix"
2018/10/08 20:08:26 [DEBUG] checking for provider in "/export/home/SIMU-LOCAL/simu/bin"
2018/10/08 20:08:26 [DEBUG] checking for provider in ".terraform/plugins/linux_amd64"
2018/10/08 20:08:26 [DEBUG] found valid plugin: "nutanix", "0.0.0", "/export/home/SIMU-LOCAL/simu/tf/terraform-provider-nutanix"
2018/10/08 20:08:26 [DEBUG] checking for provisioner in "."
2018/10/08 20:08:26 [DEBUG] checking for provisioner in "/export/home/SIMU-LOCAL/simu/bin"
2018/10/08 20:08:26 [DEBUG] checking for provisioner in ".terraform/plugins/linux_amd64"
2018/10/08 20:08:26 [INFO] command: backend <nil> is not enhanced, wrapping in local
2018/10/08 20:08:26 [INFO] backend/local: starting Apply operation
2018/10/08 20:08:26 [TRACE] Preserving existing state lineage "c3d4ac8a-1227-ec73-4f2c-ec04e584fce5"
2018/10/08 20:08:26 [TRACE] Preserving existing state lineage "c3d4ac8a-1227-ec73-4f2c-ec04e584fce5"
2018/10/08 20:08:26 [ERR] Checkpoint error: Get https://checkpoint-api.hashicorp.com/v1/check/terraform?arch=amd64&os=linux&signature=87991fa4-6ecc-57c7-77d4-cbee9a48d3f4&version=0.11.8: dial tcp 23.23.145.13:443: connect: connection refused
2018/10/08 20:08:26 [INFO] terraform: building graph: GraphTypeInput
2018/10/08 20:08:26 [TRACE] ConfigTransformer: Starting for path: []
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ConfigTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.LocalTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.OutputTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.OrphanResourceTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.OrphanOutputTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
2018/10/08 20:08:26 [TRACE] AttachResourceConfigTransformer: Beginning...
2018/10/08 20:08:26 [TRACE] AttachResourceConfigTransformer: Attach resource config request: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] Attaching resource config: &config.Resource{Mode:1, Name:"testvm", Type:"nutanix_virtual_machine", RawCount:(*config.RawConfig)(0xc4203e1c70), RawConfig:(*config.RawConfig)(0xc4203e1c00), Provisioners:[]*config.Provisioner{}, Provider:"", DependsOn:[]string(nil), Lifecycle:config.ResourceLifecycle{CreateBeforeDestroy:false, PreventDestroy:false, IgnoreChanges:[]string(nil)}}
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.AttachResourceConfigTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
2018/10/08 20:08:26 [DEBUG] Resource state not found for "data.nutanix_virtual_machine.testvm": data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.AttachStateTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.RootVariableTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
2018/10/08 20:08:26 [TRACE] ProviderConfigTransformer: Starting for path: []
2018/10/08 20:08:26 [TRACE] Attach provider request: []string{} nutanix
2018/10/08 20:08:26 [TRACE] Attaching provider config: *config.ProviderConfig{Name:"nutanix", Alias:"", Version:"", RawConfig:(*config.RawConfig)(0xc4203e1b90)}
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ProviderConfigTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.MissingProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [DEBUG] resource data.nutanix_virtual_machine.testvm using provider provider.nutanix
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.PruneProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ParentProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.graphTransformerMulti:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ModuleVariableTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.RemovedModuleTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [DEBUG] ReferenceTransformer: "data.nutanix_virtual_machine.testvm" references: []
2018/10/08 20:08:26 [DEBUG] ReferenceTransformer: "provider.nutanix" references: []
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ReferenceTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.CountBoundaryTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.TargetsTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.CloseProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.CloseProvisionerTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.RootTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
root - terraform.graphNodeRoot
  meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  provider.nutanix (close) - *terraform.graphNodeCloseProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.TransitiveReductionTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeAbstractResource
root - terraform.graphNodeRoot
  meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  provider.nutanix (close) - *terraform.graphNodeCloseProvider
2018/10/08 20:08:26 [DEBUG] Starting graph walk: walkInput
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "meta.count-boundary (count boundary fixup)"
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "provider.nutanix (close)"
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "root"
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "provider.nutanix"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "meta.count-boundary (count boundary fixup)" waiting on "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "provider.nutanix (close)" waiting on "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "root" waiting on "meta.count-boundary (count boundary fixup)"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "root" waiting on "provider.nutanix (close)"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "data.nutanix_virtual_machine.testvm" waiting on "provider.nutanix"
2018/10/08 20:08:26 [TRACE] dag/walk: dependencies changed for "meta.count-boundary (count boundary fixup)", sending new deps
2018/10/08 20:08:26 [TRACE] dag/walk: dependencies changed for "provider.nutanix (close)", sending new deps
2018/10/08 20:08:26 [TRACE] dag/walk: dependencies changed for "root", sending new deps
2018/10/08 20:08:26 [TRACE] dag/walk: dependencies changed for "data.nutanix_virtual_machine.testvm", sending new deps
2018/10/08 20:08:26 [TRACE] dag/walk: walking "provider.nutanix"
2018/10/08 20:08:26 [TRACE] vertex 'root.provider.nutanix': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.provider.nutanix': evaluating
2018/10/08 20:08:26 [TRACE] [walkInput] Entering eval tree: provider.nutanix
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInitProvider
2018-10-08T20:08:26.456+0200 [DEBUG] plugin: starting plugin: path=/export/home/SIMU-LOCAL/simu/tf/terraform-provider-nutanix args=[/export/home/SIMU-LOCAL/simu/tf/terraform-provider-nutanix]
2018-10-08T20:08:26.457+0200 [DEBUG] plugin: waiting for RPC address: path=/export/home/SIMU-LOCAL/simu/tf/terraform-provider-nutanix
2018-10-08T20:08:26.464+0200 [DEBUG] plugin.terraform-provider-nutanix: plugin address: timestamp=2018-10-08T20:08:26.463+0200 address=/tmp/plugin441897704 network=unix
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalOpFilter
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalGetProvider
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInterpolateProvider
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalBuildProviderConfig
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInputProvider
2018/10/08 20:08:26 [TRACE] root: eval: terraform.EvalNoop
2018/10/08 20:08:26 [TRACE] root: eval: terraform.EvalNoop
2018/10/08 20:08:26 [TRACE] root: eval: terraform.EvalNoop
2018/10/08 20:08:26 [TRACE] [walkInput] Exiting eval tree: provider.nutanix
2018/10/08 20:08:26 [TRACE] dag/walk: walking "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': walking
2018/10/08 20:08:26 [TRACE] dag/walk: walking "meta.count-boundary (count boundary fixup)"
2018/10/08 20:08:26 [TRACE] vertex 'root.meta.count-boundary (count boundary fixup)': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.meta.count-boundary (count boundary fixup)': evaluating
2018/10/08 20:08:26 [TRACE] [walkInput] Entering eval tree: meta.count-boundary (count boundary fixup)
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalCountFixZeroOneBoundaryGlobal
2018/10/08 20:08:26 [TRACE] [walkInput] Exiting eval tree: meta.count-boundary (count boundary fixup)
2018/10/08 20:08:26 [TRACE] dag/walk: walking "provider.nutanix (close)"
2018/10/08 20:08:26 [TRACE] vertex 'root.provider.nutanix (close)': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.provider.nutanix (close)': evaluating
2018/10/08 20:08:26 [TRACE] [walkInput] Entering eval tree: provider.nutanix (close)
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalCloseProvider
2018/10/08 20:08:26 [TRACE] [walkInput] Exiting eval tree: provider.nutanix (close)
2018/10/08 20:08:26 [TRACE] dag/walk: walking "root"
2018/10/08 20:08:26 [TRACE] vertex 'root.root': walking
2018/10/08 20:08:26 [INFO] terraform: building graph: GraphTypeValidate
2018/10/08 20:08:26 [TRACE] ConfigTransformer: Starting for path: []
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ConfigTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.LocalTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.OutputTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.OrphanResourceTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.OrphanOutputTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
2018/10/08 20:08:26 [TRACE] AttachResourceConfigTransformer: Beginning...
2018/10/08 20:08:26 [TRACE] AttachResourceConfigTransformer: Attach resource config request: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] Attaching resource config: &config.Resource{Mode:1, Name:"testvm", Type:"nutanix_virtual_machine", RawCount:(*config.RawConfig)(0xc4203e1c70), RawConfig:(*config.RawConfig)(0xc4203e1c00), Provisioners:[]*config.Provisioner{}, Provider:"", DependsOn:[]string(nil), Lifecycle:config.ResourceLifecycle{CreateBeforeDestroy:false, PreventDestroy:false, IgnoreChanges:[]string(nil)}}
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.AttachResourceConfigTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
2018/10/08 20:08:26 [DEBUG] Resource state not found for "data.nutanix_virtual_machine.testvm": data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.AttachStateTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.RootVariableTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
2018/10/08 20:08:26 [TRACE] ProviderConfigTransformer: Starting for path: []
2018/10/08 20:08:26 [TRACE] Attach provider request: []string{} nutanix
2018/10/08 20:08:26 [TRACE] Attaching provider config: *config.ProviderConfig{Name:"nutanix", Alias:"", Version:"", RawConfig:(*config.RawConfig)(0xc4203e1b90)}
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ProviderConfigTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.MissingProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [DEBUG] resource data.nutanix_virtual_machine.testvm using provider provider.nutanix
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.PruneProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ParentProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.graphTransformerMulti:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.MissingProvisionerTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ProvisionerTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.graphTransformerMulti:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ModuleVariableTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.RemovedModuleTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [DEBUG] ReferenceTransformer: "data.nutanix_virtual_machine.testvm" references: []
2018/10/08 20:08:26 [DEBUG] ReferenceTransformer: "provider.nutanix" references: []
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ReferenceTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.CountBoundaryTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.TargetsTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.CloseProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.CloseProvisionerTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.RootTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
root - terraform.graphNodeRoot
  meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  provider.nutanix (close) - *terraform.graphNodeCloseProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.TransitiveReductionTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
  provider.nutanix - *terraform.NodeApplyableProvider
meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResource
root - terraform.graphNodeRoot
  meta.count-boundary (count boundary fixup) - *terraform.NodeCountBoundary
  provider.nutanix (close) - *terraform.graphNodeCloseProvider
2018/10/08 20:08:26 [DEBUG] Starting graph walk: walkValidate
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "provider.nutanix"
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "meta.count-boundary (count boundary fixup)"
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "provider.nutanix (close)"
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "root"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "provider.nutanix (close)" waiting on "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "root" waiting on "meta.count-boundary (count boundary fixup)"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "root" waiting on "provider.nutanix (close)"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "data.nutanix_virtual_machine.testvm" waiting on "provider.nutanix"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "meta.count-boundary (count boundary fixup)" waiting on "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] dag/walk: dependencies changed for "meta.count-boundary (count boundary fixup)", sending new deps
2018/10/08 20:08:26 [TRACE] dag/walk: dependencies changed for "provider.nutanix (close)", sending new deps
2018/10/08 20:08:26 [TRACE] dag/walk: dependencies changed for "root", sending new deps
2018/10/08 20:08:26 [TRACE] dag/walk: dependencies changed for "data.nutanix_virtual_machine.testvm", sending new deps
2018/10/08 20:08:26 [TRACE] dag/walk: walking "provider.nutanix"
2018/10/08 20:08:26 [TRACE] vertex 'root.provider.nutanix': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.provider.nutanix': evaluating
2018/10/08 20:08:26 [TRACE] [walkValidate] Entering eval tree: provider.nutanix
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInitProvider
2018/10/08 20:08:26 [TRACE] root: eval: terraform.EvalNoop
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalOpFilter
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalGetProvider
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInterpolateProvider
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalBuildProviderConfig
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalValidateProvider
2018/10/08 20:08:26 [TRACE] root: eval: terraform.EvalNoop
2018/10/08 20:08:26 [TRACE] root: eval: terraform.EvalNoop
2018/10/08 20:08:26 [TRACE] [walkValidate] Exiting eval tree: provider.nutanix
2018/10/08 20:08:26 [TRACE] dag/walk: walking "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': evaluating
2018/10/08 20:08:26 [TRACE] [walkValidate] Entering eval tree: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInterpolate
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalIf
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalValidateCount
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalCountFixZeroOneBoundary
2018/10/08 20:08:26 [TRACE] [walkValidate] Exiting eval tree: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': expanding/walking dynamic subgraph
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ResourceCountTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResourceInstance
2018/10/08 20:08:26 [DEBUG] Resource state not found for "data.nutanix_virtual_machine.testvm": data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.AttachStateTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResourceInstance
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.TargetsTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResourceInstance
2018/10/08 20:08:26 [DEBUG] ReferenceTransformer: "data.nutanix_virtual_machine.testvm" references: []
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ReferenceTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResourceInstance
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.RootTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeValidatableResourceInstance
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] dag/walk: walking "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': evaluating
2018/10/08 20:08:26 [TRACE] [walkValidate] Entering eval tree: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalValidateResourceSelfRef
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalGetProvider
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInterpolate
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalValidateResource
2018/10/08 20:08:26 [TRACE] [walkValidate] Exiting eval tree: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] dag/walk: walking "meta.count-boundary (count boundary fixup)"
2018/10/08 20:08:26 [TRACE] vertex 'root.meta.count-boundary (count boundary fixup)': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.meta.count-boundary (count boundary fixup)': evaluating
2018/10/08 20:08:26 [TRACE] [walkValidate] Entering eval tree: meta.count-boundary (count boundary fixup)
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalCountFixZeroOneBoundaryGlobal
2018/10/08 20:08:26 [TRACE] [walkValidate] Exiting eval tree: meta.count-boundary (count boundary fixup)
2018/10/08 20:08:26 [TRACE] dag/walk: walking "provider.nutanix (close)"
2018/10/08 20:08:26 [TRACE] vertex 'root.provider.nutanix (close)': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.provider.nutanix (close)': evaluating
2018/10/08 20:08:26 [TRACE] [walkValidate] Entering eval tree: provider.nutanix (close)
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalCloseProvider
2018/10/08 20:08:26 [TRACE] [walkValidate] Exiting eval tree: provider.nutanix (close)
2018/10/08 20:08:26 [TRACE] dag/walk: walking "root"
2018/10/08 20:08:26 [TRACE] vertex 'root.root': walking
2018/10/08 20:08:26 [INFO] backend/local: apply calling Refresh
2018/10/08 20:08:26 [INFO] terraform: building graph: GraphTypeRefresh
2018/10/08 20:08:26 [TRACE] No managed resources in state during refresh, skipping managed resource transformer
2018/10/08 20:08:26 [TRACE] ConfigTransformer: Starting for path: []
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ConfigTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.OrphanResourceTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
2018/10/08 20:08:26 [DEBUG] Resource state not found for "data.nutanix_virtual_machine.testvm": data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.AttachStateTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
2018/10/08 20:08:26 [TRACE] AttachResourceConfigTransformer: Beginning...
2018/10/08 20:08:26 [TRACE] AttachResourceConfigTransformer: Attach resource config request: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] Attaching resource config: &config.Resource{Mode:1, Name:"testvm", Type:"nutanix_virtual_machine", RawCount:(*config.RawConfig)(0xc4203e1c70), RawConfig:(*config.RawConfig)(0xc4203e1c00), Provisioners:[]*config.Provisioner{}, Provider:"", DependsOn:[]string(nil), Lifecycle:config.ResourceLifecycle{CreateBeforeDestroy:false, PreventDestroy:false, IgnoreChanges:[]string(nil)}}
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.AttachResourceConfigTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.RootVariableTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
2018/10/08 20:08:26 [TRACE] ProviderConfigTransformer: Starting for path: []
2018/10/08 20:08:26 [TRACE] Attach provider request: []string{} nutanix
2018/10/08 20:08:26 [TRACE] Attaching provider config: *config.ProviderConfig{Name:"nutanix", Alias:"", Version:"", RawConfig:(*config.RawConfig)(0xc4203e1b90)}
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ProviderConfigTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.MissingProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [DEBUG] resource data.nutanix_virtual_machine.testvm using provider provider.nutanix
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.PruneProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ParentProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.graphTransformerMulti:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.LocalTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.OutputTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ModuleVariableTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [DEBUG] ReferenceTransformer: "provider.nutanix" references: []
2018/10/08 20:08:26 [DEBUG] ReferenceTransformer: "data.nutanix_virtual_machine.testvm" references: []
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ReferenceTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.TargetsTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.CloseProviderTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.RootTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.TransitiveReductionTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
  provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix - *terraform.NodeApplyableProvider
provider.nutanix (close) - *terraform.graphNodeCloseProvider
  data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResource
2018/10/08 20:08:26 [DEBUG] Starting graph walk: walkRefresh
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "provider.nutanix (close)"
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "provider.nutanix"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "data.nutanix_virtual_machine.testvm" waiting on "provider.nutanix"
2018/10/08 20:08:26 [TRACE] dag/walk: added edge: "provider.nutanix (close)" waiting on "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] dag/walk: dependencies changed for "data.nutanix_virtual_machine.testvm", sending new deps
2018/10/08 20:08:26 [TRACE] dag/walk: dependencies changed for "provider.nutanix (close)", sending new deps
2018/10/08 20:08:26 [TRACE] dag/walk: walking "provider.nutanix"
2018/10/08 20:08:26 [TRACE] vertex 'root.provider.nutanix': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.provider.nutanix': evaluating
2018/10/08 20:08:26 [TRACE] [walkRefresh] Entering eval tree: provider.nutanix
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInitProvider
2018/10/08 20:08:26 [TRACE] root: eval: terraform.EvalNoop
2018/10/08 20:08:26 [TRACE] root: eval: terraform.EvalNoop
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalOpFilter
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalGetProvider
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInterpolateProvider
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalBuildProviderConfig
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalOpFilter
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalConfigProvider
2018/10/08 20:08:26 [TRACE] [walkRefresh] Exiting eval tree: provider.nutanix
2018/10/08 20:08:26 [TRACE] dag/walk: walking "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': evaluating
2018/10/08 20:08:26 [TRACE] [walkRefresh] Entering eval tree: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInterpolate
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalCountCheckComputed
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalIf
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalCountFixZeroOneBoundary
2018/10/08 20:08:26 [TRACE] [walkRefresh] Exiting eval tree: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': expanding/walking dynamic subgraph
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ResourceCountTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResourceInstance
2018/10/08 20:08:26 [TRACE] OrphanResourceCount: Starting...
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.OrphanResourceCountTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResourceInstance
2018/10/08 20:08:26 [DEBUG] Resource state not found for "data.nutanix_virtual_machine.testvm": data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.AttachStateTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResourceInstance
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.TargetsTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResourceInstance
2018/10/08 20:08:26 [DEBUG] ReferenceTransformer: "data.nutanix_virtual_machine.testvm" references: []
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.ReferenceTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResourceInstance
2018/10/08 20:08:26 [TRACE] Graph after step *terraform.RootTransformer:

data.nutanix_virtual_machine.testvm - *terraform.NodeRefreshableDataResourceInstance
2018/10/08 20:08:26 [TRACE] dag/walk: added new vertex: "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] dag/walk: walking "data.nutanix_virtual_machine.testvm"
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': walking
2018/10/08 20:08:26 [TRACE] vertex 'root.data.nutanix_virtual_machine.testvm': evaluating
2018/10/08 20:08:26 [TRACE] [walkRefresh] Entering eval tree: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalSequence
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalWriteState
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalInterpolate
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalIf
2018/10/08 20:08:26 [TRACE] root: eval: terraform.EvalNoop
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalGetProvider
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalReadDataDiff
2018/10/08 20:08:26 [TRACE] root: eval: *terraform.EvalReadDataApply
data.nutanix_virtual_machine.testvm: Refreshing state...
2018/10/08 20:08:26 [ERROR] root: eval: *terraform.EvalReadDataApply, err: data.nutanix_virtual_machine.testvm: Invalid address to set: []string{"nic_list", "0", "subnet_reference_name"}
2018/10/08 20:08:26 [ERROR] root: eval: *terraform.EvalSequence, err: data.nutanix_virtual_machine.testvm: Invalid address to set: []string{"nic_list", "0", "subnet_reference_name"}
2018/10/08 20:08:26 [TRACE] [walkRefresh] Exiting eval tree: data.nutanix_virtual_machine.testvm
2018/10/08 20:08:26 [TRACE] dag/walk: upstream errored, not walking "provider.nutanix (close)"

Error: Error refreshing state: 1 error(s) occurred:

* data.nutanix_virtual_machine.testvm: 1 error(s) occurred:

* data.nutanix_virtual_machine.testvm: data.nutanix_virtual_machine.testvm: Invalid address to set: []string{"nic_list", "0", "subnet_reference_name"}



2018/10/08 20:08:26 [DEBUG] plugin: waiting for all plugin processes to complete...
2018-10-08T20:08:26.680+0200 [DEBUG] plugin.terraform-provider-nutanix: 2018/10/08 20:08:26 [ERR] plugin: plugin server: accept unix /tmp/plugin441897704: use of closed network connection
2018-10-08T20:08:26.682+0200 [DEBUG] plugin: plugin process exited:
```

**Versions (please complete the following information):**
 - OS that is executing Terraform: Macosx 10.13.6 & Debian GNU/Linux 8.9 (jessie)
 - Terraform: v0.11.8
 - Nutanix Cluster (Prism Element / AOS): 5.9
 - Nutanix Prism Central: 5.9
 - Terraform provider version (using posted RC1? Compiling yourself? If so, from Develop or Master, and what commit id? develop branch 8aae444afecbce22f4460fb62403c0ca9197c785

--- a/nutanix/data_source_nutanix_virtual_machine.go
+++ b/nutanix/data_source_nutanix_virtual_machine.go
@@ -277,16 +277,16 @@ func dataSourceNutanixVirtualMachine() *schema.Resource {
 										Type:     schema.TypeString,
 										Computed: true,
 									},
-									"name": {
-										Type:     schema.TypeString,
-										Computed: true,
-									},
 									"uuid": {
 										Type:     schema.TypeString,
 										Computed: true,
 									},
 								},
 							},
+						},
+						"subnet_reference_name": {
+							Type:     schema.TypeString,
+							Computed: true,
 						},
 					},
 				},

